mne/datasets/sleep_physionet/age.py

```diff
@@ -20,9 +20,8 @@ def fetch_data(subjects, recording=(1, 2), path=None, force_update=False,
                update_path=None, base_url=BASE_URL, on_missing='raise',
                verbose=None):  # noqa: D301
     """Get paths to local copies of PhysioNet Polysomnography dataset files.
-
     This will fetch data from the publicly available subjects from PhysioNet's
-    study of age effects on sleep in healthy subjects [1]_[2]_. This
+    study of age effects on sleep in healthy subjects :footcite:`MourtazaevEtAl1995,GoldbergerEtAl2000`. This
```

Comment:
this won't actually cause an error because we already have a `# noqa E501` comment at the end of this docstring.  But we still should constrain the line to fit:
```suggestion
    study of age effects on sleep in healthy subjects
    :footcite:`MourtazaevEtAl1995,GoldbergerEtAl2000`. This
```